### PR TITLE
webonchain will come back

### PR DESCRIPTION
**Pull request by Thykof**
image

### CODE REVIEW

**Review by thomas-senechal — APPROVED** [review]
Looks good to me
Waiting for @SlnPons approval to merge

File: web/massastation/src/i18n/en_US.json [hunk 1]
```diff
@@ -69,5 +69,9 @@
       "search": "Search Massa ecosystem",
       "explore": "Explore module store"
     }
+  },
+  "placeholder": {
+    "teaser-banner": "Web-On-Chain Evolution Incoming!",
```
Comment (SlnPons) [inline web/massastation/src/i18n/en_US.json:74]: New web interactions incoming!

File: web/massastation/src/i18n/en_US.json [hunk 2]
```diff
@@ -69,5 +69,9 @@
...
+    "teaser-web-on-chain": "Crafting a new era for web interactions..."
```
Comment (SlnPons) [inline web/massastation/src/i18n/en_US.json:75]: We're busy crafting a new era for our web on-chain feature...

**Comment by thomas-senechal** [comment]
We had a talk with @SlnPons about the placeholder sentances and decided to go with: 
> A new way to use web is coming
> We are crafting a new era for our web on-chain feature…

**Comment by thomas-senechal** [comment]
image

**Review by pivilartisant — APPROVED** [review]

**Review by peterjah — APPROVED** [review]

**Pull request merged**